### PR TITLE
Terse notes for 2021-10-21 board meeting

### PR DESCRIPTION
**Pull request by rrrutledge**

### CODE REVIEW

**Review by gruetter — APPROVED** [review]
LGTM, @rrrutledge . Thanks for the notes.

**Comment by rrrutledge** [comment]
Great!  Will try to wait for another reviewer to merge ...

File: content/about/board/2021-07-21.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: page
 show_meta: false
-title: 'Terse Notes from Board Meeting on 2021-07-21'
+title: '2021-07-21'
```
Comment (lenucksi) [inline content/about/board/2021-07-21.md:4]: Thanks! I was meaning to do that too, but didn't get to it! 🚀

**Review by lenucksi — APPROVED** [review]
This looks fine to me.

**Comment by rrrutledge** [comment]
Thanks, both!

**Pull request merged**